assets/js/custom.js

```diff
@@ -2,50 +2,65 @@ var custom = {
     loadTasks: function(numSubtasks) {
```

Comment:
General comment: this is kind of sad, maybe, since JavaScript has a type system that isn't really ideal for this kind of interface-esque design. Maybe it's worth splitting up this file into `custom-aggregate` and `custom-non-aggregate` (or some other better name) and making them both example files.